### ISSUE TITLE
gPC ROM generated using SparseGridCollocation with Gamma distribution is not working correctly 

### ISSUE THREAD

**Issue by wangcj05** [idaholab/raven#693]
--------
Issue Description
--------
##### What did you expect to see happen?
The gPC ROM generated using SparseGridCollocation with Gamma distribution should generate similar results with MonteCarlo

##### What did you see instead?
The predicted value from gPC does not agree with the Code executions, the inputs are youngsModulus, and force. The outputs are disp_x, and disp_y. 
Results from Code:
```
youngsModulus,force,disp_x,disp_y
176574.579712,-132.140792122,0.0254384584278,-0.172387649733
425755.525175,-7.85914919469,0.000627476268902,-0.0042521900282
190033.125005,-83.1397770671,0.0148717340472,-0.100780606935
210154.666871,-14.6123587696,0.00236354215596,-0.0160169091405
231284.205188,-43.5948157121,0.00640723917344,-0.0434196476773
267340.506387,-266.852951695,0.0339304257776,-0.229934780476
```
Results from gPC:
```
youngsModulus,force,disp_x,disp_y
176574.579712,-132.140792122,5.47380623359e-09,-3.70940949265e-08
425755.525175,-7.85914919469,1.40689665974e-10,-9.53405290966e-10
190033.125005,-83.1397770671,3.17405035702e-09,-2.15094433782e-08
210154.666871,-14.6123587696,4.97911911674e-10,-3.37417711342e-09
231284.205188,-43.5948157121,1.33521875082e-09,-9.04831646883e-09
267340.506387,-266.852951695,7.05075070558e-09,-4.7780503148e-08
```

If I change the Gamma distribution to Normal or using NDspline ROM with Monte Carlo, both will generate similar results compared to the code outputs. 
##### Do you have a suggested fix for the development team?
I guss the problem is scaling/normalization factors for the polynomial coefficients. 

##### Please attach the input file(s) that generate this error.  The simpler the input, the faster we can find the issue.

[tests.zip](https://github.com/idaholab/raven/files/2202323/tests.zip)

----------------
For Change Control Board: Issue Review
----------------
This review should occur before any development is performed as a response to this issue.
- [x] 1. Is it tagged with a type: defect or improvement?
- [x] 2. Is it tagged with a priority: critical, normal or minor?
- [x] 3. If it will impact requirements or requirements tests, is it tagged with requirements?
- [x] 4. If it is a defect, can it cause wrong results for users? If so an email needs to be sent to the users.
- [x] 5. Is a rationale provided? (Such as explaining why the improvement is needed or why current code is wrong.)

-------
For Change Control Board: Issue Closure
-------
This review should occur when the issue is imminently going to be closed.
- [x] 1. If the issue is a defect, is the defect fixed?
- [x] 2. If the issue is a defect, is the defect tested for in the regression test system?  (If not explain why not.)
- [x] 3. If the issue can impact users, has an email to the users group been written (the email should specify if the defect impacts stable or master)?
- [x] 4. If the issue is a defect, does it impact the latest stable branch? If yes, is there any issue tagged with stable (create if needed)?
- [x] 5. If the issue is being closed without a merge request, has an explanation of why it is being closed been provided?

**Comment by alfoa** [comment]
@wangcj05 @PaulTalbot-INL who is taking this?

**Comment by PaulTalbot-INL** [comment]
An email for the users should be drafted noting the problem and the fix.

**Comment by wangcj05** [comment]
@PaulTalbot-INL Here it is. 

Proposed Email:

Dear Users,

We recently realized that there was a defect in the SparseGridCollocation Sampler when Gamma distribution is employed as described in issue #693. This issue has been resolved with the PR#713 and the modification is currently available in the devel branch.

**Comment by alfoa** [comment]
Email accepted... and checklist passed...
